Barrier fix, Content log Fixes

MegadronA03 opened the pull request:
### Summary:
<!--- Summary of changes made in your pull request and why those changes were made -->
Fixed all content log errors.
Barriers now no more invisible.
Entity rendering changes.

---------
### Screenshots:
<!--- Screenshots of your changes (In a before-and-after format if applicable) -->
![Screenshot_20210716-153851_Minecraft](https://user-images.githubusercontent.com/56801003/126002251-39c562b5-85f4-411c-b985-26c759cc2287.jpg)

MegadronA03 committed:
Barrier invisible block fix.

Content log shader errors fixed.

MegadronA03 committed:
Added comments to changes

MegadronA03 committed:
Barrier fix

Added missing varyings parameters for renderchunck.fragment

MegadronA03 committed:
Merge pull request #1 from MegadronA03/MegadronA03-patch-1

Barrier fix

jebbyk commented:
@MegadronA03 Oh, nice, also you can add your self to developers list in mainfest.json and loading messages

MegadronA03 committed:
Update manifest.json

MegadronA03 committed:
Update loading_messages.json

MegadronA03 commented:
Added

MegadronA03 committed:
Entity underwater wobble

Improved underwater wobble effect, added env_effects.glsl for env effects, so its not only for underwater effect, so people can change effects more easily.

MegadronA03 committed:
Merge branch 'jebbyk:develop' into develop

MegadronA03 committed:
Update CHANGELOG.md

MegadronA03 committed:
Better underwater distortions

MegadronA03 committed:
Useless branch cleaning

jebbyk commented on shaders/glsl/animations.glsl:
```diff
@@ -0,0 +1,11 @@
+#define WIND_SPEED 8.0
```
@MegadronA03  are u sure this file in a correct directory? I think it should be in /glsl/options/preset

jebbyk reviewed (commented):
I suggest you to move  five last commits to separte PR. I want to release 0.13.2 today. but I have no time to test all the stuff you've added. Please leave only content log and barier fix here :-) thanx.

jebbyk committed:
Delete animations.glsl

jebbyk committed:
Update banner.vertex

jebbyk committed:
Update banner.vertex

jebbyk committed:
Update banner.vertex

jebbyk committed:
Update CHANGELOG.md

jebbyk committed:
Update entity.vertex

jebbyk committed:
Update entity.vertex

jebbyk committed:
Update entity.vertex

jebbyk committed:
Update entity.vertex

jebbyk committed:
Update entity.vertex

jebbyk committed:
Delete env_effects.glsl

jebbyk committed:
Update renderchunk.vertex

jebbyk committed:
Update banner.vertex
